src/helpers.js

```diff
@@ -1,3 +1,17 @@
+function isNulledBox(box) {
```

Comment:
I do **not** say that we should use another library. I'm saying that we should not duplicate code that other libraries provide, that our users already use.

By all means, create a SVG helper function and re-use them as much as possible. But `matches`, `camelCase` and `capitalize` might be functions that other libraries already provides.

I'm just saying that we should only make helper functions because we need them and it will slim down SVG.js. And that we should be careful not to provide the same functions that other libraries provide.